Source/engine/render/text_render.cpp

```diff
@@ -167,7 +168,7 @@ OptionalClxSpriteList LoadFont(GameFontTables size, text_color color, uint16_t r
 	const std::string_view language_code = GetLanguageCode();
 	const std::string_view language_tag = language_code.substr(0, 2);
 	if (language_tag == "zh" || language_tag == "ja" || language_tag == "ko"
-	    || (language_tag == "tr" && row == 0)) {
+		|| (language_tag == "tr" && row == 0)) {

```

Comment:
There seem to be some whitespace changes here that will fail clang-format
We use tabs for structural indentation but spaces for continuation indentation.

Reply:
will revert this change